auctions/english-auction/project/auction-contract/tests/functions/bid.rs

```diff
@@ -264,24 +263,11 @@ mod success {
         let buy_asset = nft_asset(buy_nft_contract_id, 0).await;
         let bid_asset = nft_asset(buy_nft_contract_id, 0).await;
 
-        constructor(
-            access_control,
-            &seller.nft,
-            seller_identity.clone(),
-            sell_count,
-        )
-        .await;
         mint(sell_count, &seller.nft, seller_identity.clone()).await;
-        approve(auction_identity.clone(), &seller.nft, 0).await;
-        constructor(
-            access_control,
-            &buyer1.nft,
-            buyer1_identity.clone(),
-            reserve_count,
-        )
-        .await;
+        approve(Some(auction_identity.clone()), &seller.nft, 0).await;
+
         mint(reserve_count, &buyer1.nft, buyer1_identity.clone()).await;
-        approve(auction_identity.clone(), &buyer1.nft, 0).await;
+        approve(Some(auction_identity.clone()), &buyer1.nft, 0).await;
```

Comment:
I believe the traditional Rust syntax is to just have `Some()` and `None()` as opposed to `Option::Some()`. This however is not possible in Sway.

Reply:
Perhaps, at the same time Rust follows importing in such a way that it's clear where each item is coming from. Also, I'm pretty sure `None()` is incorrect since it doesn't take any params and so it should be `None`

Reply:
Yes sorry, `None` for Rust and `None()` for Sway

Reply:
`None` in Sway is also a bug. I'm pretty sure that is meant to be fixed, if it is not already in the latest release.